src/content/docs/tutorials/javareporter/java-check-tests.md

```diff
@@ -0,0 +1,147 @@
+**java-check-tests** and run the `sync` command.  
+Here is one for UNIX/MAC users:
+```bash
+  export TESTOMATIO_URL=... && \
+  export TESTOMATIO=... && \
+  curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar && \
+  java -jar testomatio.jar sync
+```
+And this one is for Windows users:
+```cmd
+    set TESTOMATIO_URL=...&& ^
+    set TESTOMATIO=...&& ^
+    curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar&& ^
+    java -jar testomatio.jar sync
+```
+
+The `TESTOMATIO_URL` here is the server URL. In general, this will be `https://app.testomat.io` or `https://beta.testomat.io`.   
+`TESTOMATIO` is your project API key that you can get from **Testomat.io > Account > Access-Tokens**.  
+Troubleshooting: check if you haven't missed any whitespaces while editing the query.
+>**NOTE**: This command will download the jar file **testomatio.jar** to the directory from which you run the query.  
+>This file will remain in the directory and **won't be removed automatically**.
+
+After you run this command, here is what you are supposed to see:
+> **In the terminal:**
+>
+<img src="img/firstSyncConsoleResult.png" alt="First sync command run console result image" style="max-width: 47%; height: auto;">
+
+> **In the Testomat.io UI:**
+>
+<img src="img/uiImportResult.png" alt="UI import result image" style="max-width: 47%; height: auto;">
+
+> **In your test classes:**
+>
+<img src="img/codebaseDiff.png" alt="Codebase diff image after sync" style="max-width: 47%; height: auto;">
+
+If you already have the `testomatio.jar` in the directory and need to sync again, run this:
+
+```bash
+  java -jar testomatio.jar --url=... --apikey=...
+```
+
+### What it does implicitly:
+The `sync` command runs `import` and `pull-ids` consecutively after import succeeds.  
+The **import** command parses your codebase and imports it to Testomat.io.  
+The **pull-ids** command adds or updates the @TestId annotations to your test methods and related imports to the test classes.
+
+
+---
+
+## Command options
+Optionally, you can use a property for the CLI to search for tests in another directory by providing:
+- `--directory=./relative/path/from/current`. This option works for other commands as well.
+---
+
+## Other commands
+Since the `testomatio.jar` is already in your project, you can use other commands:
+> `clean-ids`
+>
+This command will remove all the IDs from test methods and related imports from the test classes.  
+`--directory` works for this command.
+If you have already removed the jar from the project, you can run the oneliner you used to sync,  
+but change `sync` to `clean-ids`.
+<br/>
+<br/>
+
+
+>`import`
+> 
+This allows you to import your codebase to Testomat.io without adding/updating the IDs.  
+`--url` and --`apikey` are required(both).  
+`--directory` is optional.
+<br/>
+<br/>
+
+As you can see, the Java-Check-Tests usage is pretty straightforward.  
```

Comment:
```suggestion
As you can see, the Java-Check-Tests usage is pretty straightforward.  
```

Never write like this

This line means nothing
This is not straightforward for some users